packages/botonic-core/jest.config.js

```diff
@@ -0,0 +1,24 @@
+// Options about JS are for compiling @botonic .js/jsx files
+module.exports = {
+  roots: ['src/'],
+  testRegex: '(/tests/.*|(\\.|/)(test|spec))\\.(js|jsx)$',
+  testPathIgnorePatterns: [
+    'lib',
+    '.*.d.ts',
+    'tests/helpers',
+    '.*.helper.js',
+    'tests/__mocks__',
+  ],
+  collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!/node_modules/'],
```

Comment:
I think that we don't have any file with this extension, but if we want to cover all Javascript scenarios, I would add also `mjs` extension. https://medium.com/passpill-project/files-with-mjs-extension-for-javascript-modules-ced195d7c84a

Reply:
thanks for the observation, I think we can add it when we have that kind of files